nx-dev/ui-pricing/src/lib/getting-started.tsx

```diff
@@ -0,0 +1,30 @@
+export function GettingStarted() {
```

Comment:
Instead of this one, can we rather use the same we use at the end of the new nx.dev page? And just delete this one
![image](https://github.com/user-attachments/assets/d6bca66d-fb82-4785-9005-5eb5b37c4da4)